plugins/mcreator-core/datalists/blocksitems.yaml

```diff
@@ -4587,16 +4587,16 @@
   readable_name: "Deepslate Bricks"
   type: block
 
-- Blocks.DEEPSLATE_BRICKS_STAIRS:
-  readable_name: "Deepslate Bricks Stairs"
+- Blocks.DEEPSLATE_BRICK_STAIRS:

```

Comment:
As this is the id used in block item selector, won't this break existing workspace?

Reply:
The 1.17 datapack generator wasn't part of 2021.2 and there's no 2021.3 snapshot yet, so I don't think a converter is needed

Reply:
Oh yes, the deepslatw is a 1.17 block. I had in mind was the new 1.16 nether stone. xD